tests/test_cli.py

```diff
@@ -46,32 +48,117 @@ def assert_expected_file(self, result, output_path):
         except:
             pass
 
-    def test_schema_convert_cli(self, runner, helpers):
-        data_model_csv_path = helpers.get_data_path("example.model.csv")
+    @pytest.mark.parametrize(
+        "output_path",
+        [
+            "tests/data/example.model.pickle",
+            "tests/data/example.model.jsonld"
+        ],
+        ids=["output_path_pickle", "output_path_jsonld"]
+    )
+    @pytest.mark.parametrize(
+        "output_type",
+        [
+            "jsonld",
+            "graph",
+            "all"
+        ],
+        ids=["output_type_jsonld", "output_type_graph", "output_type_all"]
+    )
+    def test_schema_convert_cli(self, runner, output_path, output_type):
+        model = "tests/data/example.model.csv"
+        label_type = "class_label"
+        expected = 0
 
-        output_path = helpers.get_data_path("example.model.jsonld")
+        resultOne = runner.invoke(
+            schema,
+            [
+                "convert",
+                model
+            ]
+        )
 
-        label_type = "class_label"
+        assert resultOne.exit_code == expected
+
+        resultTwo = runner.invoke(
+            schema,
+            [
+                "convert",
+                model,
+                "--output_path",
+                output_path
+            ]
+        )
+
+        assert resultTwo.exit_code == expected
+
+        resultThree = runner.invoke(
+            schema,
+            [
+                "convert",
+                model,
+                "--output_type",
+                output_type
+            ]
+        )
+
+        assert resultThree.exit_code == expected
```

Comment:
check for the creation of the file as an asssertion in addition to getting the expected output. you can also parse the file and check its contents here. Delete after the check